.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+### Summary
+
+This pull request aims to... (add your summary here)
+
+### Motivation / Context
```

Comment:
I propose to change this and the line above to make the difference between the two fields clearer.
I have the feeling what we really want is motivation.

```suggestion
### Motivation
```